hr_employee_calendar_planning/views/hr_employee_views.xml

```diff
@@ -6,10 +6,10 @@
         <field name="model">hr.employee</field>
         <field name="inherit_id" ref="hr.view_employee_form"/>
         <field name="arch" type="xml">
-            <field name="calendar_id" position="attributes">
-                <attribute name="invisible">1</attribute>
+            <field name="resource_calendar_id" position="attributes">
+                <attribute name="invisible">0</attribute>
```

Comment:
This is invisible on purpose. Don't remove it.

Reply:
Oh yes sorry, I put it visible to do some tests and forgot to change it back.